src/hotspot/share/jvmci/jvmciCompilerToVM.cpp

```diff
@@ -1577,16 +1586,18 @@ C2V_VMENTRY_NULL(jobject, iterateFrames, (JNIEnv* env, jobject compilerToVM, job
   return nullptr;
 C2V_END
 
-C2V_VMENTRY_0(int, resolveInvokeDynamicInPool, (JNIEnv* env, jobject, ARGUMENT_PAIR(cp), jint index))
-  if (!ConstantPool::is_invokedynamic_index(index)) {
-    JVMCI_THROW_MSG_0(IllegalStateException, err_msg("not an invokedynamic index %d", index));
+C2V_VMENTRY_0(int, invokeDynamicOperandToCPIndex, (JNIEnv* env, jobject, ARGUMENT_PAIR(cp), jint operand, jboolean resolve))
```

Comment:
Is `invokeDynamicOperand` the correct name here? From what I recall, the operand to an `invokedynamic` instruction is a constant pool index which is later rewritten to an `indy_index`. I think the term operand can be confusing here. 

Given the way that this method works, the operand in question is always an encoded `indy_index`.

Reply:
I've pushed a commit with better naming.